### ISSUE TITLE
Properties not in order.

### ISSUE THREAD

**Issue by samr28** [CSSLint/csslint#715]
Getting this error:

```
1: warning at line 22, col 1
Rule doesn't have all its properties in alphabetical order.
.slideTwo label {
```

Code:
```css
.slideTwo label {
  background: #fcfff4;
  background: linear-gradient(to bottom, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%);
  background: -moz-linear-gradient(top, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%); /* FF3.6+ */
  background: -ms-linear-gradient(top, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%); /* IE10+ */
  background: -o-linear-gradient(top, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%); /* Opera 11.10+ */
  background: -webkit-gradient(linear, top, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%); /* Chrome,Safari4+ */
  background: -webkit-linear-gradient(top, #fcfff4 0%, #dfe5d7 40%, #b3bead 100%); /* Chrome10+,Safari5.1+ */
  border-radius: 50px;
  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.3);
  cursor: pointer;
  display: block;
  height: 22px;
  left: 4px;
  position: absolute;
  top: 4px;
  transition: all 0.4s ease;
  width: 22px;
  z-index: 1;
  -moz-webkit-transition: all 0.4s ease;
  -moz-transition: all 0.4s ease;
  -o-transition: all 0.4s ease;
  -webkit-transition: all 0.4s ease;
}
```

I read this: https://github.com/CSSLint/csslint/issues/670 but I don't understand what they mean. If the vendor prefixes are ignored then should these be organized as if they were this:

```css
  -moz-webkit-transition: all 0.4s ease; = webkit-transition: all 0.4s ease;
  -moz-transition: all 0.4s ease; = transition: all 0.4s ease;
  -o-transition: all 0.4s ease; = transition: all 0.4s ease;
  -webkit-transition: all 0.4s ease; = transition: all 0.4s ease;
```
This is what I thought but when changing them, I still got the error. Please let me know what I'm doing wrong.
